src/widgets/WidgetMainSmall.vue

```diff
@@ -25,7 +25,11 @@
         </Button>
       </div>
       <div class="widget__close">
-        <ButtonClose icon-opacity="1" fill="var(--color-1-contrast)">
+        <ButtonClose
+          icon-opacity="1"
+          fill="var(--color-1-contrast)"
+          @click="closeWidget()"
```

Comment:
Please change this to `@click"closeWidget"`

Reply:
I'm surprised that it worked with the ().